tests/unit/AmalgamationFiling.spec.ts

```diff
@@ -49,13 +50,122 @@ describe('Amalgamation Filing', () => {
     wrapper.destroy()
   })
 
-  it('Displays expected content with a valid filing', () => {
+  it('Displays expected content with a valid filing', async () => {
     // verify content
     expect(wrapper.find('.item-header-title').text()).toBe('Amalgamation Application - Regular')
     expect(wrapper.find('.item-header-subtitle').text()).toContain('FILED AND PAID')
     expect(wrapper.find('.item-header-subtitle').text()).toContain('(filed by John Doe on Jan 1, 2021)')
     expect(wrapper.find('.item-header-subtitle').text()).toContain('EFFECTIVE as of Jan 1, 2021')
+    expect(wrapper.find('.view-details').text()).toBe('View Documents')
+    expect(wrapper.find('.hide-details').text()).toBe('Hide Documents')
 
-    // FUTURE: expand the panel and verify content
+    await wrapper.find('.view-details').trigger('click')
+    expect(wrapper.find('.completed-ia-details').exists()).toBe(true)
+    expect(wrapper.find('h4').text()).toBe('Amalgamation Complete')
+    expect(wrapper.find('p').text()).toBe('MY COMPANY has been successfully amalgamated.')
+
+    wrapper.destroy()
+  })
+})
+
+describe('Horizontal amalgamation Filing', () => {
+  let wrapper: Wrapper<AmalgamationFiling>
```

Comment:
We don't always use a type for the wrapper but it's a good idea so that `wrapper.vm` contains the correct properties/methods 👍